ui/src/designer/properties-panel.js

```diff
@@ -205,6 +205,11 @@ PropertiesPanel.prototype.render = function(element, step) {
         // Append layout html to the main div
         self.DOMObject.html(html);
 
+        // Hide any elements with layout-designer-only class, hides actions tab in Playlist editor
+        if(app.mainObjectType === 'playlist') {
```

Comment:
Linked to my prior comment - I think we'd be dynamically adding the actions grid and therefore not doing this (this might even be where we add it?)